_health-care/_js/_components/error-message.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+class ErrorMessage extends React.Component {
```

Comment:
I know I haven't been doing this, but let's start adding comments on the top of classes explaining their basic semantics and API.   For lack of any better info, I'm going to suggest we use JSDoc format:

   https://en.wikipedia.org/wiki/JSDoc

but happy to use something else if you have preferences.